packages/shared/src/hooks/log/useLogPageView.ts

```diff
@@ -1,32 +1,29 @@
-import { MutableRefObject, useContext, useEffect, useRef } from 'react';
+import { useCallback, useContext, useEffect } from 'react';
 import { useRouter } from 'next/router';
 import LogContext from '../../contexts/LogContext';
 
-export default function useLogPageView(): MutableRefObject<() => void> {
+export default function useLogPageView(): () => void {
```

Comment:
We need to make sure that moving from ref to callback doesn't cause duplicate events to be triggered